### PR TITLE
1999: Handle tags and assets when syndicating and pushing BPI content

### PR DESCRIPTION
**Pull request by rimi-itk**
http://platform.dandigbib.org/issues/1999

These changes depend on a pull request to bpi-client: https://github.com/ding2/bpi-client/pull/11

### CODE REVIEW

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: 9066fef1e7b1503d329e5c5e18afadac4c519b77
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: 9066fef1e7b1503d329e5c5e18afadac4c519b77
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

File: ding2.make [hunk 1]
```diff
@@ -425,8 +425,9 @@ projects[xautoload][version] = "5.7"
 ; Libraries
 libraries[bpi-client][destination] = "modules/bpi/lib"
 libraries[bpi-client][download][type] = "git"
-libraries[bpi-client][download][url] = "http://github.com/ding2/bpi-client.git"
-libraries[bpi-client][download][tag] = "7.x-3.0.0-rc2"
+libraries[bpi-client][download][url] = "https://github.com/rimi-itk/bpi-client.git"
+libraries[bpi-client][download][branch] = "Issue-1999"
```
Comment (JrgenGNielsen) [inline ding2.make]: makefile shouldn't point to a developer featurebranch

**Comment by rimi-itk** [comment]
Please clarify: Do you want me to add a tag to my branch and refer to this tag?

**Comment by JrgenGNielsen** [comment]
reviewed and accepted

**Pull request merged**